doc/pmon/pmon-chassis-design.md

```diff
@@ -358,6 +356,29 @@ Changes in thermalctld would follow one of the 2 approaches:
 1. Have a TemperatureUpdater class for each line-card. Each of the updater class will fetch the values for all temperature senosors of the line-card from the REDIS-DB of the line-card and update the DB on the control-card.
 2. The TemperatureUpdater class in each line-card will update the local-DB on its card as well as the global-DB on the control-card.
 
+We have chosen to take Approach 2. A GLOBAL_STATE_DB will be added in the chassis-redis instance similar to the STATE_DB in the local-redis instance.
+
+#### GLOBAL_STATE_DB Schema
+```
+key                                   = TEMPERATURE_INFO_<card-index> | <Sensor-Name>; 
+; field                               = value
+critical_high_threshold               = float;
+critical_low_threshold                = float;   
+high_threshold                        = float;
+low_threshold                         = float;
+maximum_temperature                   = float; //Maximum recorded temperature
+minimum_temperature                   = float; //Minimum recorded temperature
+temperature                           = float;
+
```

Comment:
Certain platforms/vendors have 'Voltage' and 'Current' sensors as well beside 'Temperature' sensors.
There are actions associated when these environmental crosses their respective thresholds (minor/major/critical)
Don't see them mentioned in PMON HLD. 
Can we take that into consideration?

Reply:
Added in the Future Items section